cognee/modules/graph/cognee_graph/CogneeGraphElements.py

```diff
@@ -109,8 +115,14 @@ def is_edge_alive_in_dimension(self, dimension: int) -> bool:
     def add_attribute(self, key: str, value: Any) -> None:
         self.attributes[key] = value
 
-    def get_attribute(self, key: str, value: Any) -> Union[str, int, float]:
-        return self.attributes[key]
+    def get_attribute(self, key: str) -> Optional[Union[str, int, float]]:
+        return self.attributes.get(key)
+
+    def get_node_from(self):
```

Comment:
We usually call these nodes ‘source’ and ‘destination’

Reply:
will rename them

Reply:
Done